### DISCUSSION TITLE
An error occurred handling a request for the Admin UI: Error: Prisma error: The table `main.User` does not exist in the current database

### DISCUSSION THREAD

**Discussion by MK1993** [keystonejs/keystone#8082] [Questions] [answered]
I tried to deploy a keystone app to Heroku and I did it but while I tried to open the app I got the following error: An error occurred handling a request for the Admin UI: Error: Prisma error: The table main.User does not exist in the current database. Here's a screenshot containing more details about the error: 
![Screenshot from 2022-11-08 11-58-13](https://user-images.githubusercontent.com/36052172/200549552-afcca0c0-1a84-41e3-aca7-032065830bed.png)

**Comment by NdauwaRafael** [comment]
Did you find a solution for this, I am also facing the same problem.

**Comment by designeng** [comment]
The same error. I have deleted initial database and created another one, but after `npm run build` and `npm start` I can't invoke the process of database initialisation even if I navigate browser to `/init` route.

**Reply by designeng** [reply]
And, BTW, there's should be the way for developers to restore admin user if login params was lost.

**Comment by artrodriguezcabrera** [comment]
Hoping I can bump this issue up. I am having the exact same issue trying to deploy.  Have tried the recommended fix from this other discussion, but still get the same error.

https://github.com/keystonejs/keystone/discussions/7442

**Reply by dcousens** [reply]
@artrodriguezcabrera can you elaborate on what you attempted?

**Comment by dcousens** [comment, marked as answer]
This is a duplicate of https://github.com/keystonejs/keystone/discussions/7442, please see the answer https://github.com/keystonejs/keystone/discussions/7442#discussioncomment-2521064 from @borisno2 in that discussion
